store/cachemulti/store.go

```diff
@@ -122,9 +123,15 @@ func (cms Store) GetStoreType() types.StoreType {
 // Write calls Write on each underlying store.
 func (cms Store) Write() {
 	cms.db.Write()
+	wg := sync.WaitGroup{}
+	wg.Add(len(cms.stores))
 	for _, store := range cms.stores {
-		store.Write()
+		go func(s types.CacheWrap) {
```

Comment:
most of the time, the dirty writes are small or even empty, does it justify the overhead of goroutine?

Reply:
we should get some benchmarks.